setup.py

```diff
@@ -12,6 +12,7 @@
             "data/fromTTH/fakerate/*.root",
             "data/leptonSF/*/*.root",
             "data/leptonSF/*/*.json",
+            "data/photonSF/*.root",
```

Comment:
Are there photon SFs that are included as part of this PR?

Reply:
That PR was merged, so I'm assuming this was from a git merge. Can you confirm @jlawrenc?

Reply:
@jlawrenc can you update this PR so this line doesn't show as new?